application/classes/Ushahidi/Validator/Contact/Update.php

```diff
@@ -69,18 +69,8 @@ public function valid_contact($contact, $data, $validation)
 		}
 
 		// Valid Phone?
-		// ++TODO: There's no easy to validate international numbers
-		// so just look for numbers only.
-		else if ( isset($data['type']) AND
-			$data['type'] == Contact::PHONE )
-		{
-			// Remove all non-digit characters from the number
-			$number = preg_replace('/\D+/', '', $contact);
-
-			if (strlen($number) == 0)
-			{
-				$validation->error('contact', 'invalid_phone', [$contact]);
-			}
-		}
+		// There's no easy to validate phone numbers. Even just accepting numbers
```

Comment:
We probably shouldn't be using a regex to validate phone numbers. Maybe use https://github.com/googlei18n/libphonenumber but that still won't validate "Safaricom". For now, we could use a separate validator for the receive use case?